src/containers/announcement/announcement-detail/announcement-detail.utils.ts

```diff
@@ -0,0 +1,36 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { NotionAPI } from 'notion-client';
+import { Client } from '@notionhq/client';
+
+export const notion = new NotionAPI();
+
+const { NOTION_API_KEY } = process.env;
+
+export const getAnnouncementNoteDetail = async (id: string) => {
+  const notionClient = new Client({ auth: NOTION_API_KEY });
```

Comment:
```suggestion
const notionClient = new Client({ auth: NOTION_API_KEY });

export const getAnnouncementNoteDetail = async (id: string) => {
```

클라이언트 인스턴스가 컴포넌트 리렌더링 시마다 생성되어야 하는 것일까요? 아니라면 컴포넌트 외부에서 한 번만 생성하면 될 것 같아서요~